tidb-cloud/airbyte.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Connect Airbyte with TiDB
+summary: Learn how to use Airbyte TiDB connector.
+---
+
+# Connect Airbyte with TiDB
```

Comment:
Change the title to "Replicate data with Airbyte"

Reply:
@hfxsd Thanks for your review. Done!